static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
I don't think it is supposed to be normalized here - as the equivalent of that call is like writing a `System.import('foo')` which wouldn't be normalized either?